.github/workflows/build.yaml

```
@@ -28,16 +28,16 @@ jobs:
       uses: actions/setup-node@v4
       with:
         node-version: 18.16
-        cache: yarn
+        cache: pnpm
         cache-dependency-path: |
-          yarn.lock
+          pnpm-lock.yaml
 
     - name: Run Type Check
-        uses: borales/actions-yarn@v4
+        uses: borales/actions-pnpm@v4
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 36: mapping values are not allowed in this context

(syntax-check)